svthalia/concrexit
# Decouple pushnotifications

Closes #2758.
Closes #1009. 
Closes #2755.



### Summary
This removes the dependencies from various apps on pushnotifications, by scheduling/sending notifications from signals instead of within models/views themselves.

### How to test
Steps to test the changes you made (this is kinda already done by the tests I wrote):
1. Try out things that should trigger a pushnotification.
2. Receive pushnotifications (or check that Message/ScheduledMessage objects are created).
